opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -246,5 +262,88 @@ def end(self):
             self.end_time = util.time_ns()
 
 
+def generate_span_id():
+    """Get a new random span ID.
+
+    Returns:
+        A random 64-bit int for use as a span ID
+    """
+    return random.getrandbits(64)
+
+
+def generate_trace_id():
+    """Get a new random trace ID.
+
+    Returns:
+        A random 128-bit int for use as a trace ID
+    """
+    return random.getrandbits(128)
+
+
 class Tracer(trace_api.Tracer):
-    pass
+    """See `opentelemetry.trace.Tracer`.
+
+    Args:
+        cv: The context variable that holds the current span.
+    """
+
+    def __init__(self,
+                 cv: 'contextvars.ContextVar' = _CURRENT_SPAN_CV
```

Comment:
What about using the context-propagation layer? Will that happen in the future?

Reply:
Yeah, I'll do that in another PR.